broadinstitute/pearl
# JN -513: adding new logos to ourhealth content

Adds the logos for Stanford and HMS to the home page.  Uses the bootstrap 'row-gap' class to handle the case where the flex styling wraps

![image](https://github.com/broadinstitute/juniper/assets/2800795/dd104809-80f1-43d4-b887-5097f562b2b7)

TO TEST:
1. repopulate ourhealth
2. goto https://sandbox.ourhealth.localhost:3001/
3. confirm you see all four logos
